tests/gpflow/utilities/test_deepcopy.py

```diff
@@ -36,9 +40,18 @@ def test_deepcopy_component_clears_bijector_cache_and_deecopy(module):
     input = 1.0
     _ = module(input)
     with pytest.raises(TypeError):
-        deepcopy(module)
-    module_copy = deepcopy_components(module)
+        copy.deepcopy(module)
+    module_copy = deepcopy(module)
     assert module.var == module_copy.var
     assert module.var is not module_copy.var
     module_copy.var.assign([5.0])
     assert module.var != module_copy.var
+
+
+def test_deepcopy_with_freeze():
+    module = Nested(Nested(A()))
+    module_frozen = deepcopy(module, freeze=True)
+    assert len(module.variables) == 2
```

Comment:
this looks like a test for the testing utilities - I'm not sure about including that in the test.